src/content/documentation/enterprise/enterprise-distribution.md

```diff
@@ -153,13 +131,13 @@ If you install using the Windows Registry, Firefox will not automatically update
 
 ## Firefox settings
 
-There are two settings that affect the use of alternative installation options. The `extensions.autoDisableScopes` preference controls whether add-ons are installed automatically or after user confirmation. The `extensions.enabledScopes` preference is used to disable installation from most locations. In addition to these options, the method of setting these preferences programmatically is discussed.
+Two settings affect the use of alternative installation options. The `extensions.autoDisableScopes` preference controls whether add-ons are installed automatically or after user confirmation. The `extensions.enabledScopes` preference is used to disable installation from most locations. In addition to these options, the method of setting these preferences programmatically is discussed.
 
 <section id="controlling-automatic-installations"></section>
 
 ### Controlling automatic installation
 
-The standard downloads of Firefox are configured so that sideloads using the standard extensions folder or the Windows Registry, don’t install automatically. Depending on the version of Firefox:
+The standard downloads of Firefox are configured so that loads using the standard extensions folder or the Windows Registry don’t install automatically. Depending on the version of Firefox:
```

Comment:
@mkaply as the standard extension folder relates to the sideloading process, should I remove the reference here as well?